SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version       | Supported          |
+| ------------- | ------------------ |
+| 0.8           | :white_check_mark: |
+| < 0.8         | :x:                |
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities by email to [opem@ecsim.site](mailto:opem@ecsim.site "opem@ecsim.site").
```

Comment:
don't we have a seperate email for `gopem`?

Reply:
No, we used `OPEM` email for this project. However, for unification, we can create a new email for `GOPEM`.
We will address this matter in a distinct Pull Request.